src/test/java/bio/terra/integration/AzureIntegrationTest.java

```diff
@@ -378,12 +379,22 @@ public void testSnapshotBuilder() throws Exception {
     populateOmopTable();
 
     // Test getConcepts
-    var conceptResponse = dataRepoFixtures.getConcepts(steward, datasetId, 2);
-    List<String> conceptNames =
-        conceptResponse.getResult().stream().map(SnapshotBuilderConcept::getName).toList();
-    assertThat("Correct number of concepts are returned", conceptNames.size(), equalTo(2));
-    assertThat(
-        "expected concepts are returned", conceptNames, containsInAnyOrder("concept1", "concept3"));
+    var getConceptResponse = dataRepoFixtures.getConcepts(steward, datasetId, 2);
+    List<String> getConceptNames =
+        getConceptResponse.getResult().stream().map(SnapshotBuilderConcept::getName).toList();
+    assertThat("Correct number of concepts are returned", getConceptNames.size(), equalTo(2));
```

Comment:
This is unnecessary as it's already done by `containsInAnyOrder()`. In the case where you do want to check the size of a collection, it's better to use `hasSize(2)` instead.